__tests__/pet.test.js

```diff
@@ -0,0 +1,179 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Fido')).toBeInstanceOf(Object);
+    });
+    it ('sets the name property', () => {
```

Comment:
Its a convention to always leave a clear line between each of your `it` blocks, it just makes the test file easier to read as a whole